specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -15,39 +15,36 @@
 - [Alternatives](#alternatives)
 
 ## Summary
-
-<!--
-Short summary of the proposal. It will be used as user-focused
-documentation such as release notes or a (customer facing) development roadmap.
-The tone and content of the `Summary` section should be
-useful for a wide audience.
--->
+Create an open source frontend for the VDK.
+The frontend will let customers, create,delete deploy and generally managed nad browse their jobs.
+For frontend is written in an extensible format to easily allow other teams to build extensions.
 
 ## Glossary
-<!--
-Optional section which defines terms and abbreviations used in the rest of the document.
--->
+data-pipelines: This is the npm project of the bootstrapping codebase it contains all the vdk specific frontend code
+shared-components, @vdk/shared: This is a set of components that are used within the data-pipelines project but are generic and could be re-used by other projects
 
 ## Motivation
-<!--
-It tells **why** do we need X?
-Describe why the change is important and the benefits to users.
-Explain the user problem that need to be solved.
--->
+1. Improve usability
+   1. With a UI customers will be able to use VDK much quicker and much more proficiently because UIs are easier to understand and become familiar with than CLI tools
+2. increase VDK adoption
```

Comment:
I think the motivation section should be from the point of view of the user. It's written in the https://github.com/vmware/versatile-data-kit/edit/main/specs/NNNN-template/README.md: 

```
Describe why the change is important and the benefits to users.
Explain the user problem that need to be solved [not your problem]
```